docs/src/pages/[platform]/getting-started/troubleshooting/shared/vite-polyfill.mdx

```diff
@@ -0,0 +1,40 @@
+
+**1.** Add the following script tag to the `index.html` file right before the `</body>` tag. This will only run on the client side and will polyfill Node globals.
+
+```html
+  <script>
+    window.global = window;
+    window.process = {
+      env: { DEBUG: undefined },
+    }
+    var exports = {};
+  </script>
+</body>
+```
+
+**2.** Update the `vite.config.ts` (or `vite.config.js`) and add a resolve alias inside the `defineConfig({})` as seen below.
+
+```ts
+export default defineConfig({
+  plugins: [react()],
+  resolve: {
+      alias: [
+      {
+        find: './runtimeConfig',
+        replacement: './runtimeConfig.browser', // ensures browser compatible version of AWS JS SDK is used
+      },
+    ]
+  }
+})
+```
+
+**3.** (Optional for TS users) If you run into TypeScript strict errors from `aws-amplify` you may need to update the `tsconfig.json` file with the following config:
+
+```json
+  "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
I think `skipLibCheck` is auto-generated by the template, but we do need to add `allowJs: true`

Reply:
It's autogenerated and set to `false`, which is why the workaround for `aws-amplify` TS errors requires setting this to `true`.

Reply:
Would you double check that? I just created a new app and got `"skipLibCheck": true,`

Reply:
You're right, I'm seeing that now as well. I guess we don't need this anymore, I'll remove it now!